### DISCUSSION TITLE
Resubmit Requests

### DISCUSSION THREAD

**Discussion by cazar86** [sct/overseerr#1936] [Help]
Was doing some cleaning in Radarr and unmonitored all content before deleting old items. By doing this I accidently unmonitored the requested content from overseerr that hasn't been added yet. Is there a job to resubmit requests?

**Comment by i-am-at0m** [comment]
> Was doing some cleaning in Radarr and unmonitored all content before deleting old items. By doing this I accidently unmonitored the requested content from overseerr that hasn't been added yet. Is there a job to resubmit requests?

I am also curious on this because I lost my Sonarr instance and had to rescan everything in. I don't know what was monitored and I don't know what things that haven't come out yet were in there.

**Comment by christensenjairus** [comment]
I'm also looking for the answer to this question!

**Reply by i-am-at0m** [reply]
I was told in IRC I could basically write a script that uses the API to interrogate each request in overseerr and compare it's status to Sonarr, but that was the extent of the help I got. Unfortunately, there isn't a way of doing this baked-in, because they don't see it as 'normal' flow, so you'd have to do something custom.

 I don't know enough python to just bang out a script for this purpose, and haven't had the free time to muddle through it, so I've just been dealing with individual complaints of "hey I requested this and it's out, why isn't it in Plex yet?" By checking the request, going to Sonarr or Radarr, realizing the item is missing and then adding it as monitored content and triggering a search. Overseerr then detects that it's been fulfilled and completes the request.

**Comment by asherflynt** [comment]
Anyone ever figure this out? I had to reinstall my arrs and now I have to manually add the old requests. This is kind of ridiculous.

**Reply by c11umw** [reply]
This might not help you now depending on your scenario, but in the arrs there's an automated backup created of the DB and config.
If you still have access to the old file structure then you should be able to grab the last backup and import it into the new instance.

What I do with ours is copy the backup from each arr as part of my monthly backup processes so that even if something goes *pop* then there's very little lost to get it all back up and running with a few clicks.

**Comment by dannymcc** [comment]
For anyone experiencing this issue, I have written a simple script to query the Overseer API and then resubmit all requests.

https://github.com/dannymcc/overseer-rerequester
